release.yaml

```diff
@@ -10,21 +10,24 @@ fleet-crd:
   - 103.1.1+up0.9.1-rc.5
 harvester-cloud-provider:
   - 103.0.1+up0.2.3
+  - 103.0.2+up0.2.3
```

Comment:
In the previous commit, I'm seeing 103.0.1 as the package.yaml version, is this leftover from something else?